From 7a1de6f2d6a804c642bc4cc07c67d39b53367d5c Mon Sep 17 00:00:00 2001
From: Erfan Abdi <erfangplus@gmail.com>
Date: Sun, 31 Mar 2024 07:52:15 +0000
Subject: [PATCH] bp: Make it visible for OnDevicePersonalization

We added flatbuffers dependency in tensorflow and
that caused linker errors in packages/modules/OnDevicePersonalization.

OnDevicePersonalization (lib)
  |----> federated-compute
           |----> tensorflow
                    |----> flatbuffers (we added and need to be visible to the target)

Change-Id: I8bbf550b7d6041bc962c0ecac613cbed1ee65636
---

diff --git a/Android.bp b/Android.bp
index 2e603bb..4a99d6d 100644
--- a/Android.bp
+++ b/Android.bp
@@ -128,6 +128,7 @@
         "//apex_available:platform",
         "com.android.btservices",
         "com.android.extservices",
+        "com.android.ondevicepersonalization",
     ],
     min_sdk_version: "30",
 }
